amespahdbpythonsuite/tests/test_mcfitted.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+"""
+test_mcfitted.py
+
+Test the mcfitted.py module.
+"""
+
+import pytest
+from os.path import exists
+import matplotlib.pyplot as plt
+
+from pkg_resources import resource_filename
+
+from amespahdbpythonsuite.amespahdb import AmesPAHdb
+from amespahdbpythonsuite import observation, mcfitted
+
+
+@pytest.fixture(scope="module")
+def test_mcfitted():
+    xml = "resources/pahdb-theoretical_cutdown.xml"
+    db = AmesPAHdb(
+        filename=resource_filename("amespahdbpythonsuite", xml),
+        check=False,
+        cache=False,
+        update=False,
+    )
+    uids = [18, 73, 726, 2054, 223]
```

Comment:
Are these all the uids in the file? In that case, does a ```db.gettransitionsbuid(-1)``` work?

Reply:
It returns a Transitions(self.uids=[-1],shift=0.0), while now it returns Transitions(self.uids=[18, 73, 726, 2054, 223],shift=0.0).